src/main/java/syncsquad/teamsync/controller/MainWindowController.java

```diff
@@ -90,12 +103,31 @@ public MainWindowController(Stage primaryStage, Logic logic) {
         closeButton.setGraphic(closeIcon);
         closeButton.getStyleClass().addAll(Styles.BUTTON_ICON, Styles.FLAT, Styles.ACCENT);
 
-        this.mainVBox.setOnMousePressed((MouseEvent event) -> {
+        FontIcon minimizeIcon = new FontIcon(Material2MZ.MINIMIZE);
```

Comment:
Add a TODO here to group the window-related buttons (minimise, maximise, close) into one func